queries.md

```diff
@@ -1,25 +1,85 @@
 # Database Queries
 
 ## find all customers that live in London. Returns 6 records.
+select * from customers where city is 'London'
+
 
 ## find all customers with postal code 1010. Returns 3 customers.
+select * from customers where PostalCode like '1010'
```

Comment:
both will work but the LIKE operator should really only be used when searching for something with incomplete data/data that can contain _some_ of the string you supply - -since you know exactly which postal code you're trying to match, I'd use IS instead